src/action.js

```diff
@@ -150,7 +150,7 @@ const showLintResults = async ([from, to]) => {
     // this would be a good place to implement the setNeutral() when it's eventually implimented.
     // for now it can pass with a check mark.
     console.log('Passing despite errors ✅')
-    setOutput(`You have commit messages with errors\n\n${formattedResults}`)
+    console.log(formattedResults)
```

Comment:
What about putting something like this instead? That way the last seen we see is the success emoji
```
console.log(`You have commit messages with errors\n\n${formattedResults}`)
console.log('Passing despite errors ✅')
```